src/app/Schedule/SchedulePage.tsx

```diff
@@ -34,13 +34,30 @@ export default function SchedulePage({ currentShow }: SchedulePageProps) {
   const headerHeight = useHeaderHeight();
 
   const [schedule, setSchedule] = useState<ScheduleResponse | null>(null);
+  const [currentShowTitle, setCurrentShowTitle] = useState<string | undefined>(
+    currentShow,
```

Comment:
I actually think `currentShow` is never passed to this component. So we can remove this component's interface object, and set this to type `<string>`